### PR TITLE
missing dependency

### PR DESCRIPTION
**Pull request by ugocupcic**

### CODE REVIEW

File: lwr_hw/package.xml [hunk 1]
```diff
@@ -31,9 +32,10 @@
   <run_depend>roscpp</run_depend>
   <run_depend>controller_manager</run_depend>
   <run_depend>kdl_parser</run_depend>
+  <brun_depend>joint_limits_interface</run_depend>
```
Comment (gavanderhoorn) [inline lwr_hw/package.xml]: typo: `Brun_depend`.

**Comment by ugocupcic** [comment]
@gavanderhoorn sorry my bad!

**Comment by carlosjoserg** [comment]
A Shadowed contribution, thanks!

**Comment by ugocupcic** [comment]
:ghost:

**Pull request merged**